### ISSUE TITLE
Remove empty rows from data

### ISSUE THREAD

**Issue by Aryllen** [Sage-Bionetworks/dccvalidator#428]
There has been a problem recently with metadata files having extra, empty rows, which then gives failures in dccvalidator (incomplete required columns, for example). This isn't necessarily a bug, but it does make for undesired behavior. I don't know if we can really require people to know how to remove the extra rows that were most likely added by accident when converting the Excel template to csv. Instead, we should just remove the empty rows ourselves before doing the checks.

**Comment by karawoo** [comment]
I wonder if we should consider adding an extra check for empty rows. If we remove them in dccvalidator before checking, they'll still be in the files that get uploaded. Not a huge deal, but could cause confusion if people are reusing the files and getting unexpected missing data. Then again, if we add another check it won't prevent the existing ones from failing over the incomplete columns, which is confusing (if technically correct).

**Comment by Aryllen** [comment]
Yeah, it's annoying to have extra empty rows when they upload the metadata. I am not sure it's necessarily simple to have them remove the extra rows, though. I can imagine that many people will also remove the newline at the end, which then also messes up how it's read in. There's a trade-off between data users having to deal with empty rows (_should_ be simple for people using the data) and trying to explain how to fix it to less savvy contributors who might break the file more. 🤷‍♀️

**Comment by karawoo** [comment]
> I am not sure it's necessarily simple to have them remove the extra rows, though

That's a good point, I'm not sure I'd even know how to do it if I weren't using R. Select a bunch of empty rows in excel and click delete? I have no idea if that actually does anything, lol.
